docs/2_1_patching.md

```diff
@@ -8,16 +8,26 @@ The following pages will guide you through using ReVanced Manager to patch apps.
 2. Tap the + button in the bottom right corner
 3. Choose an app to patch[^1]
 4. Tap on the version of the app you want to patch[^2]
```

Comment:
You have to select from storage unless you have downloader plugins installed, which is not mentioned here.